find_coordinator_response.go

```diff
@@ -0,0 +1,121 @@
+package sarama
+
+import (
+	"net"
+	"strconv"
+	"time"
+)
+
+type FindCoordinatorResponse struct {
+	Version         int16
+	ThrottleTime    time.Duration
+	Err             KError
+	ErrMsg          *string
+	Coordinator     *Broker
+	CoordinatorID   int32  // deprecated: use Coordinator.ID()
+	CoordinatorHost string // deprecated: use Coordinator.Addr()
+	CoordinatorPort int32  // deprecated: use Coordinator.Addr()
```

Comment:
Can we take this opportunity to remove these, since this is technically a new struct?

Reply:
Good call. Done.